charts/dapr/charts/dapr_operator/templates/dapr_operator_service.yaml

```diff
@@ -49,6 +49,7 @@ metadata:
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
 {{- if .Values.webhookService.annotations }}
+  annotations:
```

Comment:
I think this is still under the annotations from above on line 11
```suggestion
```

Reply:
Sorry. Not sure what you mean

Reply:
I mean this `annotations` line you added can be removed since its already on line 11 above I believe, so the webhook service annotations are already under that field so we don't need to add `annotations` here. Hopefully that makes sense?

Reply:
Sorry, still not sure. Line 11 is a different resource, we want annotations on both, no?

Reply:
I think this is fine. I do think line 53 on the `toYaml` needs a dash.
```
{{ - toYaml .Values.webhookService.annotations | indent 4}}
```

Pretty sure Ale commented on this to be fixed on all of these.

Reply:
also, if we have the toYaml line, then we don't need the range loop like the service above has?

Reply:
oh, I misread it being a diff resource. yeah I agree now